articles/quickstart/native/ios-swift/_includes/_login_centralized.md

```diff
@@ -1,12 +1,21 @@
-The first step in adding authentication to your iOS application is to provide a way for your users to log in. The fastest, most secure, and most feature-rich way to do this with Auth0 is to use the [login page](/hosted-pages/login).
+The [Auth0 hosted login page](/hosted-pages/login) is the easiest way to set up authentication in your application. We recommend using the Auth0 hosted login page for the best experience, best security and the fullest array of features.
```

Comment:
don't use best twice

Reply:
This has been worded this way and approved for other quickstarts, see: https://auth0.com/docs/quickstart/spa/angular2#add-authentication-with-auth0

Reply:
For consistency then let's leave it alone.